static/css/partials/_header.scss

```diff
@@ -19,40 +19,40 @@
   width: 100%;
 
   .logo {
-    @include respond-to("small") {
-      max-width: $grid-unit * 10;
+    @media (max-width: 768px) {
```

Comment:
please try to avoid directly using media queries in the styles in favor of respond-to breakpoints here
https://github.com/mozilla-services/screenshots/blob/master/static/css/partials/_layout.scss
https://github.com/mozilla-services/screenshots/blob/master/static/css/partials/_mixins.scss#L18